src/css/ng-wig.css

```diff
@@ -194,4 +194,22 @@
   font-family: sans-serif;
   font-size: 14px;
   line-height: 24px;
+}
+
+.form-control {
+  display: block;
+  width: 100%;
+  height: 34px;
+  padding: 6px 12px;
+  font-size: 14px;
+  line-height: 1.42857143;
+  color: #555;
+  background-color: #fff;
+  background-image: none;
+  border: 1px solid #ccc;
+  border-radius: 4px;
```

Comment:
let's do border: 0? Much more consistent with other toolbar styles, see for yourself:
![image](https://cloud.githubusercontent.com/assets/6132126/8241202/9e4e0d30-1610-11e5-9e6b-2eff7d895375.png)